### DISCUSSION TITLE
Fields as modules

### DISCUSSION THREAD

**Discussion by nbenn** [BristolMyersSquibb/blockr#88] [Ideas]
In addition to supporting fields as standard shiny "input" objects, we could support fields as shiny modules.

Open questions:

- can we somehow make such "module-fields" behave like "standard input fields"?
- could a field consist of multiple shiny "input entries"?
- how valuable would such an extension be? we already support two ways of bringing in external logic (using the "fields framework" and overriding block-level methods)

cc @christophsax @DivadNojnarg

**Comment by christophsax** [comment]
A very crude experiment that distinguishes between 'module based fields' and 'input based fields': https://github.com/blockr-org/blockr/tree/experiment/complex-field. As discussed today, we can keep it on hold for now.

**Comment by christophsax** [comment]
No more fields...
